tests/data/python38.py

```diff
@@ -11,6 +11,14 @@ def starred_yield():
     yield "value1", *my_list
 
 
+# all right hand side expressions allowed in regular assignments are now also allowed in
+# annotated assignments
+a : Tuple[ str, int] = "1", 2
+a: List [ str] = *x
```

Comment:
Looks like technically this is a syntax error, but it's allowed in the cpython grammar. Black should probably not be used to validate syntax though, so it's probably fine :)

Reply:
Yeah, it's probably fine. As long we don't add parenthesizes to any of the above, the case of the following won't happen :)

> * invalid python
> * Black accepts it
> * Black generates valid python at the end

https://github.com/psf/black/issues/921#issuecomment-509653493